### PR TITLE
Identify RCP pages in the list table screen

### LINKED ISSUE #1617: Identify RCP pages in the list table screen
Some plugins, like WooCommerce, identify their pages in the Pages > All Pages list view.

Example, note the "Cart Page" and "Checkout Page" hints after the linked title:
image

I find this a useful hint when logged into customer sites, and think it could be useful for us to do the same.

We can use the `display_post_states` filter to do this.

### PR DESCRIPTION
**Pull request by tnorthcutt**
Closes #1617

### CODE REVIEW

File: includes/admin/admin-actions.php [hunk 1]
```diff
@@ -27,4 +27,32 @@ function rcp_process_actions() {
...
+ * Denotes the various RCP pages as such in the pages list table.
+ *
+ * @param array $post_states An array of post display states.
+ * @param WP_Post $post The current post object.
+ */
+function rcp_display_post_states( $post_states, $post ) {
+	$rcp_options = get_option( 'rcp_settings' );
+	$pages       = array(
+		'registration_page' => 'Registration Page',
+		'redirect'          => 'Redirect Page',
```
Comment (pippinsplugins) [inline includes/admin/admin-actions.php]: I think this should be `Registration Success Page`.

File: includes/admin/admin-actions.php [hunk 2]
```diff
@@ -27,4 +27,32 @@ function rcp_process_actions() {
...
+ */
+function rcp_display_post_states( $post_states, $post ) {
+	$rcp_options = get_option( 'rcp_settings' );
+	$pages       = array(
+		'registration_page' => 'Registration Page',
+		'redirect'          => 'Registration Success Page',
+		'account_page'      => 'Account Page',
+		'edit_profile'      => 'Edit Profile Page',
+		'update_card'       => 'Update Card Page'
+	);
```
Comment (mindctrl) [inline includes/admin/admin-actions.php]: I think these page names need to be translatable.

Comment (tnorthcutt) [reply includes/admin/admin-actions.php]: Good point, thanks!

File: includes/admin/admin-actions.php [hunk 3]
```diff
@@ -27,4 +27,32 @@ function rcp_process_actions() {
...
-add_action( 'admin_init', 'rcp_process_actions' );
\ No newline at end of file
+add_action( 'admin_init', 'rcp_process_actions' );
+
+/**
+ * Denotes the various RCP pages as such in the pages list table.
+ *
+ * @param array $post_states An array of post display states.
+ * @param WP_Post $post The current post object.
+ */
```
Comment (nosegraze) [inline includes/admin/admin-actions.php]: Can you please add `@return` to the PHPDocs?

Comment (tnorthcutt) [reply includes/admin/admin-actions.php]: Err, it got merged before I got a chance to do that – what's your preference on how to do that now?

Comment (mindctrl) [reply includes/admin/admin-actions.php]: I fixed it before merging.

Comment (tnorthcutt) [reply includes/admin/admin-actions.php]: alright-alright-alright

**Pull request merged**